domain/objectstore/state/types.go

```diff
@@ -33,8 +32,9 @@ type dbMetadataPath struct {
 // object metadata.
 func (m dbMetadata) ToCoreObjectStoreMetadata() coreobjectstore.Metadata {
 	return coreobjectstore.Metadata{
-		Hash: m.Hash,
-		Path: m.Path,
-		Size: m.Size,
+		Hash256:     m.Hash256,
+		Hash512_384: m.Hash512_384,
+		Path:        m.Path,
+		Size:        m.Size,
 	}
```

Comment:
Worth killing this while you're here?

Reply:
Happy for you to sweep back around and clean this up.